go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/google/exposure-notifications-verification-server
 
 go 1.14
 
+replace github.com/google/exposure-notifications-server => ../exposure-notifications-server
```

Comment:
Will need to update this after merging https://github.com/google/exposure-notifications-server/pull/957

Reply:
pls update to v0.7.0